app/views/votes/_toggle_like.html.erb

```diff
@@ -3,8 +3,15 @@
   liked = user_signed_in? && current_user.voted_for?(votable)
 %>
 
-<%= form_tag vote_path, method: (liked ? :delete : :post) do %>
+<%= form_tag vote_path, method: (liked ? :delete : :post), class: "like-form" do %>
   <%= text_field_tag 'id', votable.id, hidden: true %>
   <%= text_field_tag 'type', votable.class.to_s.downcase, hidden: true %>
-  <%= submit_tag '👍 ' + votable.get_likes.size.to_s, class: "btn btn-default btn-sm pull-left #{"active" if liked}" %>
+  <% if votable.class.to_s == "Comment" %>
+    <%= button_tag(type: 'submit', class: "btn btn-link btn-xs #{"inactive" unless liked}") do %>
+      <%= votable.get_likes.size.to_s %>
+      <span class="glyphicon glyphicon-triangle-top"></span>
+    <% end %>
+  <% else %>
+    <%= submit_tag '👍 ' + votable.get_likes.size.to_s, class: "btn pull-left btn-default btn-sm #{"active" if liked}" %>
+  <% end %>
```

Comment:
We should not be switching this based on the votable class. That is an antipattern: https://sourcemaking.com/refactoring/replace-conditional-with-polymorphism

Since this seems to be purely a style difference, please see if you can find a way to display a different glyph using CSS.